src/pretalx/orga/views/submission.py

```diff
@@ -388,6 +388,10 @@ def form_valid(self, form):
                 return self.get(self.request, *self.args, **self.kwargs)
             messages.success(self.request, _('The submission has been updated!'))
         if form.has_changed():
+            # handle slot_count change:
+            if 'slot_count' in form.changed_data and 'slot_count' in form.initial:
```

Comment:
When is 'slot_count' not in form.initial?

Reply:
every-time a submission is newly created from the orga backend.
i added this check after i got an error during the test-creation..